README.md

```diff
@@ -4,13 +4,13 @@
   <img src="https://user-images.githubusercontent.com/16392542/77208906-224aa500-6aba-11ea-96bd-e81806074030.png" width="350">
 </div>
 
-## AutoML Toolkit for Deep Learning
+## AutoML Toolkit for Tabular, Text, and Vision
```

Comment:
I'd change to: "for Text, Image, and Tabular Data" to be grammatically correct.

Reply:
added